web-ui/package.json

```diff
@@ -29,6 +29,7 @@
     "angularfire2": "5.0.0-rc.4",
     "core-js": "^2.4.1",
     "firebase": "4.9.0",
+    "@firebase/app-types": "^0.1.0",
```

Comment:
That's actually not needed, since typings are included in the `firebase` dependency.

Also...
* Typing should be added to `devDependencies` (= needed for development / building), not `dependencies` (= included in the build bundles).
* Dependencies must not be added manually.  
Dependencies should be installed using yarn (`yarn add <dependency>` or `yarn add -D <devDependency>`) Otherwise the `yarn.lock` file does not get updated which is responsible for making sure the exact same dependency versions are installed in every single environment.

Reply:
I see why you added that dependency, though the warning seems invalid to me, since the package was installed (as dependency of `@firebase/app`).

```
± la node_modules/@firebase
total 0
drwxr-xr-x  5 svenf 41305271 160 Feb  2 13:17 app
drwxr-xr-x  5 svenf 41305271 160 Feb  2 13:17 app-types <<<
drwxr-xr-x  7 svenf 41305271 224 Feb  2 13:17 auth
drwxr-xr-x  4 svenf 41305271 128 Feb  2 13:17 auth-types
drwxr-xr-x  5 svenf 41305271 160 Feb  2 13:17 database
drwxr-xr-x  4 svenf 41305271 128 Feb  2 13:17 database-types
drwxr-xr-x  9 svenf 41305271 288 Feb  2 13:17 firestore
drwxr-xr-x  4 svenf 41305271 128 Feb  2 13:17 firestore-types
drwxr-xr-x  5 svenf 41305271 160 Feb  2 13:17 messaging
drwxr-xr-x  4 svenf 41305271 128 Feb  2 13:17 messaging-types
drwxr-xr-x  9 svenf 41305271 288 Feb  2 13:17 polyfill
drwxr-xr-x  5 svenf 41305271 160 Feb  2 13:17 storage
drwxr-xr-x  4 svenf 41305271 128 Feb  2 13:17 storage-types
drwxr-xr-x 12 svenf 41305271 384 Feb  2 13:17 util
drwxr-xr-x  6 svenf 41305271 192 Feb  2 13:17 webchannel-wrapper
```

Reply:
This is actually a bug with yarn, I'd prefer keeping it as it removes the warning.